src/main/java/me/lokka30/microlib/events/PlayerMoveXYZEvent.java

```diff
@@ -13,41 +13,44 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
+ * This event fires when a player has moved to a different X, Y and/or Z coordinate.
+ * <p>
+ * For example, moving from x=1 to x=2 will fire the event, and, moving from x=1.1 to x=1.7 will also fire the event. However, if the player rotates their head only, the event will not fire. To reiterate, this event only fires if a player has actually move on the X/Y/Z axis. <p> Using this class in your plugin can increase its performance by not running code every time a player moves their head ever so slightly, since minor movements like these fire PlayerMoveEvent,
+ * so running lots of code on PlayerMoveEvent may have a poor impact on performance.
+ *
  * @author lokka30
+ * @version 3.0.0
+ * @apiNote This event only fires if MicroLib is installed as a plugin on the server. The event does not fire if MicroLib is shaded into the resource!
  * @see org.bukkit.event.player.PlayerMoveEvent
  * @see PlayerMoveFullXYZEvent
- * @since v3.0.0
- * <p>
- * WARNING: This event only fires if MicroLib is installed
- * as a plugin on the server. The event does not fire if
- * MicroLib is shaded into the resource!
- * <p>
- * This event fires when a player has moved to a different
- * X, Y and/or Z coordinate.
- * For example, moving from x=1 to x=2 will fire the event,
- * and, moving from x=1.1 to x=1.7 will also fire the event.
- * However, if the player rotates their head only, the event
- * will not fire. To reiterate, this event only fires if
- * a player has actually move on the X/Y/Z axis.
- * <p>
- * Using this class in your plugin can increase its performance
- * by not running code every time a player moves their head ever
- * so slightly, since minor movements like these fire PlayerMoveEvent,
- * so running lots of code on PlayerMoveEvent may have a poor impact
- * on performance.
+ * @see org.bukkit.event.Cancellable
+ * @see org.bukkit.event.Event
+ * @since 3.0.0
  */
 @SuppressWarnings("unused")
 public class PlayerMoveXYZEvent extends Event implements Cancellable {
 
     /* HandlerList */
     private static final HandlerList HANDLERS = new HandlerList();
 
+    /**
+     * Gets handlers.
```

Comment:
Same suggestions apply here as I did with the other event class. as with the rest of the javadocs in this class.